.mvn/maven.config

```diff
@@ -1,2 +1,3 @@
 -Pconsume-incrementals
 -Pmight-produce-incrementals
+-Dchangelist.format=%d.v%s
```

Comment:
this is wrong. and produces versions that are just 100% string qualifiers in Maven world.
maven related tooling will not work as expected.
Idellay use `%d-v%s`

Reply:
https://github.com/jenkinsci/incrementals-tools/issues/29 FTR